android-project/app/src/main/java/org/libsdl/app/SDLAudioManager.java

```diff
@@ -95,7 +95,7 @@ protected static int[] open(boolean isCapture, int sampleRate, int audioFormat,
         }
 
         if (audioFormat == AudioFormat.ENCODING_PCM_FLOAT) {
-            int minSDKVersion = (isCapture ? 23 /* Android 6.0 (M) */ : 21 /* Android 5.0 (LOLLIPOP) */);
+            int minSDKVersion = (records ? 23 /* Android 6.0 (M) */ : 21 /* Android 5.0 (LOLLIPOP) */);
```

Comment:
recording? Maybe do a search for "records"?

Reply:
Hmm, maybe I only searched in c and h files. I'll check.

Reply:
Yeah, it was a perl one-liner that only did `*.[ch]` ...I've now fixed up the Java (and C++ for WinRT) that it missed.